plugins/pur/core/src/test/java/com/pentaho/di/purge/RepositoryCleanupUtilTest.java

```diff
@@ -23,36 +23,35 @@
 import com.sun.jersey.api.client.filter.HTTPBasicAuthFilter;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.mockito.MockedStatic;
+import org.pentaho.di.repository.pur.AttributesMapUtil;
 
 import java.util.Base64;
 
 import static org.junit.Assert.assertEquals;
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.anyString;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.Mockito.doCallRealMethod;
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
 import static org.mockito.Mockito.when;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
-import static org.powermock.reflect.Whitebox.getInternalState;
-import static org.powermock.reflect.Whitebox.setInternalState;
+//import static org.pentaho.test.util.InternalState.setInternalState;
+//import static org.pentaho.test.util.InternalState.getInternalState;
 
-@RunWith( PowerMockRunner.class )
-@PowerMockIgnore( "jdk.internal.reflect.*" )
-@PrepareForTest( Client.class )
+//@RunWith( PowerMockRunner.class )
+//@PowerMockIgnore( "jdk.internal.reflect.*" )
+//@PrepareForTest( Client.class )
 public class RepositoryCleanupUtilTest {
 
   @Test
   public void authenticateLoginCredentials() throws Exception {
     RepositoryCleanupUtil util = mock( RepositoryCleanupUtil.class );
     doCallRealMethod().when( util ).authenticateLoginCredentials();
 
-    setInternalState( util, "url", "http://localhost:8080/pentaho" );
-    setInternalState( util, "username", "admin" );
-    setInternalState( util, "password", "Encrypted 2be98afc86aa7f2e4bb18bd63c99dbdde" );
+  //  setInternalState( util, "url", "http://localhost:8080/pentaho" );
+  //  setInternalState( util, "username", "admin" );
+  //  setInternalState( util, "password", "Encrypted 2be98afc86aa7f2e4bb18bd63c99dbdde" );
```

Comment:
I've been using ReflectionTestUtils.setField from spring-test as a drop-in replacement for the powermock whitebox stuff, and it's worked fine so far. I haven't seen `org.pentaho.test.util.InternalState.setInternalState` before, but since it's commented out, I'm assume it wasn't working right -- might be worth giving the spring one a shot here.